data/data/baremetal/main.tf

```diff
@@ -20,6 +20,7 @@ module "bootstrap" {
 module "masters" {
   source = "./masters"
 
+  ironic_uri     = var.ironic_uri
```

Comment:
I think this shouldn't be here, we removed it in our repo but not from openshift/installer yet

Reply:
Ah Ok, will fix and re-test

Reply:
Thanks, fixed